src/micrate.cr

```diff
@@ -17,10 +17,13 @@ module Micrate
 
   def self.up(db, migrations_path = DEFAULT_MIGRATIONS_PATH, migrations_table_suffix = "")
     all_migrations = migrations_by_version(migrations_path)
-
     current = dbversion(db, migrations_table_suffix)
-    target = all_migrations.keys.sort.last
-    migrate(all_migrations, current, target, db, migrations_table_suffix)
+    if !all_migrations.keys.empty?
+      target = all_migrations.keys.sort.last
+      migrate(all_migrations, current, target, db, migrations_table_suffix)
+    else
+      puts "No migrations found #{migrations_path}"
```

Comment:
Would you mind changing these to use the current logger instance instead of `puts`?

When using the CLI interface, logged messages are displayed in STDOUT, but logger gives more flexibility if using the `Micrate` module directly.

Also [this may be nitpicking, but] what about "... in #{migrations-path}"? :-)